feature-libs/order/components/order-history/order-history.component.html

```diff
@@ -17,9 +17,9 @@
         <h4 *ngIf="type.replenishmentOrder">
           {{ 'orderHistory.replenishmentHistory' | cxTranslate }}
         </h4>
-        <h3 *ngIf="!type.replenishmentOrder">
+        <h2 *ngIf="!type.replenishmentOrder">
```

Comment:
Please mention about this change in 5.0 migration doc.

Reply:
Added